docs/schema.md

```diff
@@ -48,45 +48,26 @@ Note that GHC requires all of `DEnum`, `DRecord`, `FieldDef`, and so forth to be
 
 As discussed in the introduction, `mu-schema` has been developed with some common schema formats in mind. Instead of writing the type-level schemas by hand, you can also import your [Protocol Buffers](https://developers.google.com/protocol-buffers/) schemas.
 
-The most common case is that your schema lives in an external file, maybe shared with other components of your system. To declare that we want the file to be pre-processed before compilation, we use a GHC feature called a *quasi-quote*. Be careful with the ending of the quasi-quote, which is a weird combination `|]`.
+The idea is that your schema lives in an external file, so you can share it with other components of your system. To declare that we want the file to be pre-processed before compilation, we use a GHC feature called `TemplateHaskell`, hence the initial line starting with `language`.
 
 ```haskell
-{-# language QuasiQuotes #-}
+{-# language TemplateHaskell #-}
 
-type ExampleSchema = [protobufFile|path/to/file.proto|]
+import Mu.Quasi.ProtoBuf
+
+protobuf "ExampleSchema" "path/to/file.proto"
```

Comment:
Here is `protobuf` or `protobufFile`? 🤔 I didn't know you could do this without QuasiQuotes! 💥

Reply:
One of the changes due to the new annotation format is that everything now uses `TemplateHaskell`, quasi-quotation is no longer supported.